sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3431,6 +3444,12 @@ object functions {
 
   /**
    * Returns whether a predicate holds for one or more elements in the array.
+   * {{{
+   *   df.select(exists(col("i"), _ % 2 === 0))
+   * }}}
+   *
+   * @param column the input array column
+   * @param f col => predicate, the Boolean predicate to check the input column
```

Comment:
nit: I would just call it `the boolean predicate` instead of `the Boolean predicate` in this file but no big deal.

Reply:
It's intentionally fixed by #27382.

Reply:
okie. that's fine.